### ISSUE TITLE
Add "Permission" to project manager

### ISSUE THREAD

**Issue by filhocf** [kanboard/kanboard#3245]
### Actual behaviour

I attribute for a user the flag of "project manager" to he create projects, but he can't add his team. Today, only admin role can add users to projects. 

Giving this role for some managers, they can see other projects from other owners, that isn't a behavior expected.

### Expected behaviour

That "project manager" role can add his members team in his project.

**Comment by fguillot** [comment]
There is a role "application manager" which different from "project manager": https://kanboard.net/documentation/roles

**Comment by filhocf** [comment]
I understand that already this role, but IMHO, have a gap between what have in the project and the need in project management.

Think that we have a "administrator", that can see all. Ok. He can access all, see the problems, etc.

I and you are managers. I make my projects, like you, but I wait to add my team for my projects, without see your projects. For you, is the same thing. You add your contributors, without the need that I see what you are doing.

I believe that isn't necessary that the admin add people in my or your projects. We can do it. ;-)

Did you follow me?

**Comment by Sorasma** [comment]
Good point @filhocf  , and maybe this feature will also be back into interest https://github.com/kanboard/kanboard/issues/992#issuecomment-298234563 . Will solve lots of problems with delegating a team or just a member to a specific task.

**Comment by filhocf** [comment]
@fguillot, now I could pay attention for this question. Looking your reference, we have:

Role | Description
-- | --
Administrator | Access to everything
Manager | Can create team projects but cannot change application settings
User | Can create private projects only

Now, I got a new user, called "colaborar". I wait that this user can open a new project for his team and add new collaborators into it.

![image](https://user-images.githubusercontent.com/952596/28432899-20f73bbe-6d60-11e7-96dc-6b8def84bdd1.png)

When I try to edit, as "colaborar" user, I got this page:

![image](https://user-images.githubusercontent.com/952596/28433002-8f537906-6d60-11e7-97f4-5de726676849.png)

If you attempt to "Permissions and ownership" section, haven't space to add new members. If is possible add this function in this place for manager role, we have the solution.
